src/main/java/com/adobe/aem/compgenerator/utils/ComponentUtils.java

```diff
@@ -94,13 +95,36 @@ private void createClientLibs() {
         try {
             if (generationConfig.getOptions().isHasJs() || generationConfig.getOptions().isHasCss()) {
                 createFolderWithContentXML(clientLibDirPath, Constants.TYPE_SLING_FOLDER);
+
+                String clientLibSiteDirPath = clientLibDirPath + "/site";
```

Comment:
We should generate this folder only if `isHasCss()` or `isHasJs()`

Reply:
Line 96 (3 lines above here) is handling that functionality.

Reply:
ah sorry :D